tekton/cd/pruner/overlays/oci-ci-cd/pruner-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-pruner-default-spec
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/part-of: tekton-pruner
+    pruner.tekton.dev/config-type: global
+data:
+  global-config: |
+    enforcedConfigLevel: namespace
+    ttlSecondsAfterFinished: 604800
+    successfulHistoryLimit: 30
+    failedHistoryLimit: 10
+    namespaces:
+      tekton-ci:
```

Comment:
These are all the namespaces we clean today: https://github.com/tektoncd/plumbing/tree/main/tekton/cronjobs/nightly_oci/cleanup
For tekton-ci, we keep a combined history of 200 today.